hasher-matcher-actioner/hmalib/common/configs/tests/test_tx_collab_config.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+from threatexchange.exchanges.clients import ncmec
+from threatexchange.exchanges.clients.ncmec.hash_api import NCMECEnvironment
+
+from threatexchange.exchanges.impl.fb_threatexchange_api import (
+    FBThreatExchangeCollabConfig,
+)
+from threatexchange.exchanges.impl.ncmec_api import (
+    NCMECCollabConfig,
+)
+
+
+from hmalib.common.models.tests.ddb_test_common import HMAConfigTestBase
+from hmalib.common.configs import tx_collab_config
+
+
+class TXCollabConfigsTestCase(HMAConfigTestBase, unittest.TestCase):
+    TABLE_NAME = f"test-config-{__name__}"
+
+    def test_get_no_collab_config(self):
+        with self.fresh_dynamodb():
+            self.assertEqual([], tx_collab_config.get_all_collab_configs())
+
+    def test_write_one_config(self):
+        with self.fresh_dynamodb():
+            tx_collab_config.create_collab_config(
+                FBThreatExchangeCollabConfig(
+                    name="Threatcollaboration Something", privacy_group=1212312
+                )
+            )
+            all_configs = tx_collab_config.get_all_collab_configs()
+            self.assertEqual(1, len(all_configs))
+            self.assertEqual("Threatcollaboration Something", all_configs[0].name)
```

Comment:
I'm considering enforcing that names are only lower_under in the CLI to make them easier to parse from from bash, and have considered various conventions to make it easier to spot that something was a collab config name.

Having started this road, do you have similar feelings about constraining the name?

Reply:
bash has a need to quote strings with spaces. The same problem does not exist on web forms and the like. I don't see HMA users running into an issue with this.